packages/import-bundle/src/index.js

```diff
@@ -124,6 +174,21 @@ export async function importBundle(bundle, options = {}, powers = {}) {
   }
 }
 
+/**
+ * A utility function for producing test bundles, which are not seerializable
+ * as JSON or passable.
+ * @param {Record<PropertyKey, unknown>} exports
+ */
+export const bundleTestExports = exports => {
+  const symbols = Object.getOwnPropertySymbols(exports);
+  symbols.length > 0 &&
+    Fail`exports must not have symbol-named properties, got: ${symbols.map(String).join(', ')}`;
```

Comment:
Let's be flexible and allow a `Symbol.toStringTag` own prop so that a module namespace object can be used directly.